# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmBCClass.h
# Commit: 13ee9674ee35
# Full Hash: 13ee9674ee356acb36f3badb5ab2043d3f608eb1
# Author: Lars T Hansen <lhansen@mozilla.com>
# Date: 2021-10-18 16:09:29
# Regressor Bug: 1727084
# File Overlap Count: 1
# Description:
#   Bug 1727084 - Memory64 - Bulk memory operations. r=yury
#   
#   Implement all bulk memory operations on 64-bit memories.  Only
#   out-of-line paths are supported for memory.copy and memory.fill, for
#   now.
# ==============================================================================

diff -r b658cfe4b173 -r 13ee9674ee35 js/src/wasm/WasmBCClass.h
--- a/js/src/wasm/WasmBCClass.h	Mon Oct 18 09:58:14 2021 +0000
+++ b/js/src/wasm/WasmBCClass.h	Mon Oct 18 09:58:14 2021 +0000
@@ -1439,13 +1439,13 @@
   [[nodiscard]] bool emitAtomicXchg(ValType type, Scalar::Type viewType);
   [[nodiscard]] bool emitMemInit();
   [[nodiscard]] bool emitMemCopy();
-  [[nodiscard]] bool emitMemCopyCall(uint32_t lineOrBytecode);
-  void emitMemCopyInline();
+  [[nodiscard]] bool memCopyCall(uint32_t lineOrBytecode);
+  void memCopyInlineM32();
   [[nodiscard]] bool emitTableCopy();
   [[nodiscard]] bool emitDataOrElemDrop(bool isData);
   [[nodiscard]] bool emitMemFill();
-  [[nodiscard]] bool emitMemFillCall(uint32_t lineOrBytecode);
-  void emitMemFillInline();
+  [[nodiscard]] bool memFillCall(uint32_t lineOrBytecode);
+  void memFillInlineM32();
   [[nodiscard]] bool emitTableInit();
   [[nodiscard]] bool emitTableFill();
   [[nodiscard]] bool emitTableGet();